### ISSUE TITLE
VPP may fail on IPRouteAddDel

### ISSUE THREAD

**Issue by glazychev-art** [networkservicemesh/sdk-vpp#348]
### Description 
Found on VPP v21.06

### Build
See `TestRunHealSuite`, forwarders.
https://github.com/networkservicemesh/integration-k8s-aws/actions/runs/1143649866

### Logs
```
2021-08-18T16:07:53.028621844Z Aug 18 16:07:53.025[37m [DEBU] [cmd:/bin/forwarder] [duration:104.721µs] [isAdd:true] [nh.address:c0a8:4001::] [prefix:2600:1f16:b1b:c900::/64] [swIfIndex:1] [vppapi:IPRouteAddDel] [0mcompleted
2021-08-18T16:07:53.028632966Z Aug 18 16:07:53.025[37m [DEBU] [cmd:/bin/forwarder] [duration:117.795µs] [isAdd:true] [nh.address:c0a8:4001::] [prefix:fe80::/64] [swIfIndex:1] [vppapi:IPRouteAddDel] [0mcompleted
2021-08-18T16:07:53.028637427Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: received signal SIGSEGV, PC 0x7f4b47d1b8d7, faulting address 0x0
2021-08-18T16:07:53.028642147Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #0  0x00007f4b4795c028 0x7f4b4795c028
2021-08-18T16:07:53.028646139Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #1  0x00007f4b478b93c0 0x7f4b478b93c0
2021-08-18T16:07:53.02865037Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #2  0x00007f4b47d1b8d7 ip6_sas_by_sw_if_index + 0x127
2021-08-18T16:07:53.028654346Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #3  0x00007f4b47d2cf81 ip6_neighbor_probe_dst + 0x31
2021-08-18T16:07:53.028658328Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #4  0x00007f4b47d211f4 ip_neighbor_update + 0x114
2021-08-18T16:07:53.028662283Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #5  0x00007f4b48070621 adj_nbr_add_or_lock + 0xe1
2021-08-18T16:07:53.028666335Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #6  0x00007f4b48062d08 0x7f4b48062d08
2021-08-18T16:07:53.028670247Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #7  0x00007f4b4806578f fib_path_resolve + 0x2bf
2021-08-18T16:07:53.028674226Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #8  0x00007f4b4805fafb 0x7f4b4805fafb
2021-08-18T16:07:53.02867821Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #9  0x00007f4b48060571 fib_path_list_create + 0x321
2021-08-18T16:07:53.028682264Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #10 0x00007f4b4805b3b2 0x7f4b4805b3b2
2021-08-18T16:07:53.028686228Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #11 0x00007f4b480597e7 fib_entry_src_action_path_swap + 0x97
2021-08-18T16:07:53.028690187Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #12 0x00007f4b480578d9 fib_entry_update + 0x59
2021-08-18T16:07:53.028694192Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #13 0x00007f4b4804d5c5 fib_table_entry_update + 0x215
2021-08-18T16:07:53.028698305Z Aug 18 16:07:51.898[36m [INFO] [cmd:vpp] [0mvpp[12]: #14 0x00007f4b4806db05 fib_api_route_add_del + 0x65
2021-08-18T16:07:53.088952873Z Aug 18 16:07:53.088[31m [ERRO] [cmd:/bin/forwarder] [0msignal: aborted
2021-08-18T16:07:53.091385555Z panic: error: context canceled
2021-08-18T16:07:53.091403808Z 
2021-08-18T16:07:53.091408611Z goroutine 1 [running]:
2021-08-18T16:07:53.091412586Z github.com/networkservicemesh/cmd-forwarder-vpp/internal/vppinit.Must(...)
2021-08-18T16:07:53.091416736Z 	/build/internal/vppinit/vppinit.go:68
2021-08-18T16:07:53.091420902Z main.main()
2021-08-18T16:07:53.091424767Z 	/build/main.go:176 +0x3085
```
[forwarder-vpp-87qv4.log](https://github.com/networkservicemesh/sdk-vpp/files/7011683/forwarder-vpp-87qv4.log)
[TestRunHealSuite.zip](https://github.com/networkservicemesh/sdk-vpp/files/7011688/TestRunHealSuite.zip)

**Comment by glazychev-art** [comment]
Found a problem: created a topic - https://lists.fd.io/g/vpp-dev/topic/adding_new_route_via_ipv6/85020362?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,85020362

**Comment by glazychev-art** [comment]
There are a couple of patches, that solve the problem:
https://gerrit.fd.io/r/c/vpp/+/33558 - merged
https://gerrit.fd.io/r/c/vpp/+/33568 - not merged yet

**Comment by glazychev-art** [comment]
PR for govpp - https://github.com/edwarnicke/govpp/pull/36

**Comment by edwarnicke** [comment]
@glazychev-art I've merged edwarnicke/govpp#36
